pyscriptjs/src/remote_interpreter.ts

```diff
@@ -84,18 +86,21 @@ export class RemoteInterpreter extends Object {
      * path.
      */
     async loadInterpreter(config: AppConfig, stdio: Stdio): Promise<void> {
+        const syncify_maybe = x => (this.useWorker ? x.syncify() : x);
```

Comment:
It will be nice to avoid this with same-thread syncify =)

Reply:
done ;)